packages/sync/integration_test/utils/graphql.queries.js

```diff
@@ -40,16 +40,16 @@ export const UPDATE_TASK = gql`
 `;
 
 export const ONLINE_ONLY = gql`
-  mutation onlineOnly($id: ID!) @onlineOnly {
-    onlineOnly(id: $id) {
+  mutation onlineOnly($id: ID!) {
+    onlineOnly(id: $id) @onlineOnly {
```

Comment:
Is this test enabled now?

Reply:
No, for some reason it still fails. I've created item for it in test plan, so that this is tested manually.